POC/ColorPicker/ColorPicker/IconHelper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Interop;
+
+namespace ColorPicker
+{
+    // Source: https://stackoverflow.com/questions/2341230/removing-icon-from-a-wpf-window
+    // and https://www.wpftutorial.net/RemoveIcon.html
+    public class IconHelper
+    {
+        [DllImport("user32.dll")]
+        static extern int GetWindowLong(IntPtr hwnd, int index);
+
+        [DllImport("user32.dll")]
+        static extern int SetWindowLong(IntPtr hwnd, int index, int newStyle);
+
+        [DllImport("user32.dll")]
+        static extern bool SetWindowPos(IntPtr hwnd, IntPtr hwndInsertAfter,
+            int x, int y, int width, int height, uint flags);
+
+        [DllImport("user32.dll")]
+        static extern IntPtr SendMessage(IntPtr hwnd, uint msg,
+            IntPtr wParam, IntPtr lParam);
```

Comment:
I don't think this function is used, may want to remove.

Reply:
Good catch!